CHANGELOG.md

```diff
@@ -21,6 +22,15 @@
 - [v0.5.0](#v050---2020-08-06)
 <!-- END TOC -->
 
+-------------------------------------------------
+## v0.20.2 - 2022-05-10
+
+### Changed
+- Increased fluentd `FOR` alerts to 30m, should decrease number of false-positive alerts.
+
+### Fixed
+
+- Issue where users couldn't do `POST` or `DELETE` requests to alertmanager via service proxy
```

Comment:
I think you should move this section in between v0.20.1 and v0.21.0 for it to be inline with the toc.

Reply:
Done!!